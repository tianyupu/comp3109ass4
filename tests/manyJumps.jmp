  i = 1;
  s = 0;
  goto L10
L1:
  i = i + 1;
  goto L2;
L2:
  return i;
L3:
  i = i + 1;
  goto L1;
L4:
  i = i + 1;
  goto L3;
L5:
  goto L4;
L6:
  goto L5;
L7:
  i = i + 1;
  goto L6;
L8:
  i = i + 1;
  goto L7;
L9:
  i = i + 1;
  goto L8;
L10:
  i = i + 1;
  goto L9;
