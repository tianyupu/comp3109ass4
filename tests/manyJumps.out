  i = 1;
  s = 0;
  goto L10
L10:
  i = i + 1;
  goto L9;
L9:
  i = i + 1;
  goto L8;
L8:
  i = i + 1;
  goto L7;
L7:
  i = i + 1;
  goto L4;
L4:
  i = i + 1;
  goto L3;
L3:
  i = i + 1;
  goto L1;
L1:
  i = i + 1;
  return i;
